contracts/test-hs/src/Lorentz/Contracts/Swaps.hs

```diff
@@ -0,0 +1,122 @@
+-- | Lorentz bindings for the swaps contract.
+module Lorentz.Contracts.Swaps
+  ( module Lorentz.Contracts.Swaps
+  ) where
```

Comment:
This looks a bit off.

Reply:
You mean, it should be indented with more spaces?

Reply:
No I mean, why does the module export itself?
There should be an export list for its content.

Reply:
This is an explicit way to say "just export everything". I thought, like, here we trivially export the entire content, since these are mere bindings that should not define anything smart.

In case any not-to-be-exported thing suddenly appears here, I'll put a proper export list there, but now it only seems to cause an unnecessary duplication.